stdlib/public/runtime/SymbolInfo.cpp

```diff
@@ -0,0 +1,97 @@
+//===--- ImageInspectionWin32.cpp - Win32 image inspection ----------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#include "SymbolInfo.h"
+
+#if defined(_WIN32) && !defined(__CYGWIN__)
+#define WIN32_LEAN_AND_MEAN
+#define NOMINMAX
+#include <Windows.h>
+#include <DbgHelp.h>
+#elif SWIFT_STDLIB_HAS_DLADDR
+#include <dlfcn.h>
+#endif
+
+#include "ImageInspection.h"
+
+using namespace swift;
+
+const char *swift::SymbolInfo::getFileName() const {
```

Comment:
I wish that we used the standard style - `file_name()` or `filename()`.

Reply:
Well, snake-case would not be consistent with the style of the rest of the runtime. "filename" and "fileName" are inconsistently used, but it does look like "filename" is more common at the runtime level. "get" as a prefix is standard here too. So I can change it to "getFilename".

Reply:
I think that the change is likely not worth it.  The wishful thinking is sufficient I think :).

Reply:
I changed it to `getFilename()` in the latest revision.